dtfs-central-api/api-tests/v1/tfm/amendments/tfm-amendments-post.api-test.js

```diff
@@ -51,10 +52,26 @@ describe('POST TFM amendments', () => {
         .put({ dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS, dealId, auditDetails: generatePortalAuditDetails(MOCK_PORTAL_USER._id) })
         .to('/v1/tfm/deals/submit');
 
-      const { body } = await api.post().to(`/v1/tfm/facilities/${newId}/amendments`);
+      const { body } = await api.post({ auditDetails: generateTfmAuditDetails(MOCK_TFM_USER._id) }).to(`/v1/tfm/facilities/${newId}/amendments`);
       expect(body).toEqual({ amendmentId: expect.any(String) });
     });
 
+    it('should return 400 if no auditDetails provided', async () => {
```

Comment:
this is where test helpers would really save us time -- do we need tests for`put` too for audit details? happy to pair if you want a hand writing these!